{
    "type": "bundle",
    "id": "bundle--fe4d5c48-abe6-4c9d-9efb-baf8f13915ef",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--1fc6936c-0dfe-472a-b1d9-27c187c8ddb3",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2022-09-20T00:08:31.10119Z",
            "modified": "2022-09-20T00:08:31.10119Z",
            "name": "CVE-2022-38880",
            "description": "The d8s-urls for python, as distributed on PyPI, included a potential code-execution backdoor inserted by a third party. The affected version is 0.1.0.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-38880"
                }
            ]
        }
    ]
}